src/content/getting-started/architecture/networkplugin-syncer/_index.en.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Networkplugin syncer"
+---
+
+The **_networkplugin-syncer_** interfaces to the K8s network plugin subsystem
+in any necessary way. Currently, it's only deployed for _OVNKubernetes_,
+see the [OVNKubernetes architecture section](./ovn-kubernetes/) for more details.
+
+The networkplugin-syncer receives the detected K8s network plugin as
+the `SUBMARINER_NETWORKPLUGIN` environment variable. This is configured by the
+_submariner-operator_ when creating the networkplugin-syncer deployment.
```

Comment:
Yes, but... I wanted to let them know that as part of our architecture the network plugin syncer does not detect, it just receives that input somehow. I can hide some detail but I want to le them know about that fact.

Reply:
Again I don't think it's relevant where and how it gets detected in a high-level architecture doc. We pass many env vars to the processes but we don't go into that detail here, eg passing the broker info to the engine.  Just trying to be consistent and keep it high-level.  @nyechiel WDYT?

Reply:
@mangelajo @tpantelis I think the challenge here is that we are trying to capture both user-facing/high level docs as well as developer docs at the same place. Maybe we could keep this Architecture section high level (to Tom's point, I don't think we have low level implementation details documented for other components at the moment) and have a separate section  under https://submariner.io/development/?

Reply:
Yeah, but in this case it's different, because, for example, in the route-agent (this same architecture tree) we provide very low level details of what's being done. As it will have multiple handlers based on that I think it's important to explain a bit at least, somewhere (not only the source code).

Please note that I have removed the exact mention of the env var. But I have been more precise about the detection.